packages/video_player/video_player_web/pubspec.yaml

```diff
@@ -29,5 +29,5 @@ dev_dependencies:
   pedantic: ^1.8.0
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.8.0 <3.0.0"
```

Comment:
I am changing this here just to be consistent with the platform interface (the platform interface requires it because of `pigeon v0.1.7`, which annotates it on a file in preparation of null safety.